engine/apps/slack/scenarios/slack_channel_integration.py

```diff
@@ -78,27 +90,28 @@ def save_thread_message_for_resolution_note(self, slack_user_identity, payload):
             )
             if len(text) > 2900:
                 if slack_thread_message.added_to_resolution_note:
-                    return self._slack_client.api_call(
+                    self._slack_client.api_call(
                         "chat.postEphemeral",
                         channel=channel,
                         user=slack_user_identity.slack_id,
                         text=":warning: Unable to update the <{}|message> in Resolution Note: the message is too long ({}). "
                         "Max length - 2900 symbols.".format(permalink, len(text)),
                     )
-                else:
-                    return
+                return
             slack_thread_message.text = text
             slack_thread_message.save()
 
         except ResolutionNoteSlackMessage.DoesNotExist:
             if len(text) > 2900:
-                return self._slack_client.api_call(
+                self._slack_client.api_call(
                     "chat.postEphemeral",
                     channel=channel,
                     user=slack_user_identity.slack_id,
                     text=":warning: The <{}|message> will not be displayed in Resolution Note: "
                     "the message is too long ({}). Max length - 2900 symbols.".format(permalink, len(text)),
                 )
+                return
```

Comment:
the return value from this function is never used. With that in mind, lets keep it consistent and always return `None`